jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenRawConfiguration.java

```diff
@@ -69,7 +71,9 @@ public AuthProperty getToAuth() {
 
   @Override
   public Iterable<String> getToTags() {
-    return jibPluginConfiguration.getTargetImageAdditionalTags();
+    Set<String> tags = jibPluginConfiguration.getTargetImageAdditionalTags();
```

Comment:
I actually preferred doing this check in `JibPluginConfiguration`. I intended `MavenRawConfiguration` to be a super-thin adapter between `RawConfiguration` and `JibPluginConfiguration` without any processing or validation. Doing the validation in `JibPluginConfiguration` may be weird as well, but given the state of `MavenRawConfiguration`, at least we can keep `MavenRawConfiguration` untainted.

That said, I've been thinking config value validation is a mess currently. It's being done here and there haphazardly for different configs.

Reply:
I agree that `JibPluginConfiguration` is the better place: it also knows whether the value came from the `jib.to.tags`  system property or `<jib><to><tags>` element.  I started it there but testing became messy.